Extensions/xkit_patches.js

```diff
@@ -138,7 +138,70 @@ XKit.extensions.xkit_patches = new Object({
 	},
 
 	patches: {
-		"7.9.0": function() {},
+		"7.9.0": function() {
+			XKit.tools.Nx_XHR = function(details) {
```

Comment:
Shouldn't this still be in 7.8.2 since the current distributed version is 7.8.2?

Reply:
putting it in 7.9.0's patches is just the lazy way, means I don't have to put it into xkit.js (yet). also i just noticed i forgot the version bump whoops

Reply:
:+1: